ch.ethz.eventb.utils.sdk/feature.xml

```diff
@@ -15,7 +15,7 @@
 <feature
```

Comment:
should update year to 2021 in copyright header

Reply:
Fixed in commit ab872f1e6fd24630f61aeda36aa29f1c2d693185.